metabrainz/templates/index/datasets.html

```diff
@@ -0,0 +1,86 @@
+{% extends 'base.html' %}
+
+{% block title %}{{ _('Datasets') }} - MetaBrainz Foundation{% endblock %}
+
+{% block content %}
+  <h1 class="page-title">{{ _('The MetaBrainz Datasets') }}</h1>
+
+  <p> We ask <a href="{{ url_for('users.account_type') }}">commercial users</a> to <a href="{{ url_for('users.account_type') }}">support us</a> in order to help fund the creation and maintenance of these datasets. </p>
+
+<p>
+  The MetaBrainz Foundation provides free, open access to various types of useful data gathered and verified by volunteers. Here, we list and describe the various datasets we maintain.
```

Comment:
I would prefer it if all the text in the html is not in one line, makes it easier to read in text editors and in the github review interface.

Reply:
This goes for all descriptions. We generally try to keep lines around a 100-120 columns long.